tests/roxctl/bats-tests/local/roxctl-netpol-analyze-development.bats

```diff
@@ -111,6 +106,114 @@ teardown() {
+}
+
+@test "roxctl-development analyze netpol generates dot connlist output" {
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml"
+  echo "Writing connlist to ${ofile}" >&3
+  run roxctl-development analyze netpol "${test_data}/np-guard/netpols-analysis-example-minimal" --output-format=dot
+  assert_success
+
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  assert_output --partial '"0.0.0.0-255.255.255.255" [label="0.0.0.0-255.255.255.255" color="red2" fontcolor="red2"]'
+}
+
+@test "roxctl-development analyze netpol generates csv connlist output" {
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml"
+  echo "Writing connlist to ${ofile}" >&3
+  run roxctl-development analyze netpol "${test_data}/np-guard/netpols-analysis-example-minimal" --output-format=csv
+  assert_success
+
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  assert_output --partial '0.0.0.0-255.255.255.255,default/frontend[Deployment],TCP 8080'
+}
+
+@test "roxctl-development analyze netpol skips unsupported openshift resources" {
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/asset-cache-deployment.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/asset-cache-route.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/frontend-netpols.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/webapp-deployment.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/webapp-route.yaml"
+  echo "Writing connlist to ${ofile}" >&3
+  run roxctl-development analyze netpol "${test_data}/np-guard/security-frontend-demo"
+  assert_success
+
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  assert_output --partial 'skipping object with type: Route'
+}
+
+@test "roxctl-development analyze netpol generates focused to workload connlist output" {
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/backend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/frontend.yaml"
+  assert_file_exist "${test_data}/np-guard/netpols-analysis-example-minimal/netpols.yaml"
+  echo "Writing connlist to ${ofile}" >&3
+  run roxctl-development analyze netpol "${test_data}/np-guard/netpols-analysis-example-minimal" --focus-workload=backend
+  assert_success
+
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  # partial here is used to filter the WARN and INFO messages
+  assert_output --partial 'default/backend[Deployment] => default/backend[Deployment] : All Connections
```

Comment:
add assertion that a line from the full output (without `focus-workload`) is not part of the output here